src/containers/tutor-home-page/language-step/LanguageStep.jsx

```diff
@@ -1,16 +1,62 @@
+import React, { useState } from 'react'
 import Box from '@mui/material/Box'
+import { Typography, Autocomplete, TextField } from '@mui/material'
 
 import { styles } from '~/containers/tutor-home-page/language-step/LanguageStep.styles'
 import img from '~/assets/img/tutor-home-page/become-tutor/languages.svg'
 
+import { languagesMock } from '../subjects-step/constants'
+
+const languages = languagesMock.map((lang) => lang.name)
+
 const LanguageStep = ({ btnsBox }) => {
+  const [selectedLanguage, setSelectedLanguage] = useState(null)
+  const [displayedLanguages, setDisplayedLanguages] = useState(
+    languages.slice(0, 6)
+  )
+
+  const handleScroll = (event) => {
+    const listboxNode = event.currentTarget
+    if (
+      listboxNode.scrollTop + listboxNode.clientHeight >=
+      listboxNode.scrollHeight
+    ) {
+      setDisplayedLanguages((prev) => [
+        ...prev,
+        ...languages.slice(prev.length, prev.length + 6)
+      ])
+    }
+  }
+
   return (
     <Box sx={styles.container}>
       <Box sx={styles.imgContainer}>
         <Box component='img' src={img} sx={styles.img} />
       </Box>
       <Box sx={styles.rigthBox}>
-        <Box>Language here</Box>
+        <Typography variant='h6'>
+          Please select the language in which you would like to study and
+          cooperate.
+        </Typography>
+
+        <Autocomplete
+          ListboxProps={{
+            style: { maxHeight: 200, overflow: 'auto' },
+            onScroll: handleScroll
+          }}
+          filterOptions={(options, state) =>
+            options.filter((option) =>
+              option.toLowerCase().includes(state.inputValue.toLowerCase())
+            )
+          }
+          onChange={(event, newValue) => setSelectedLanguage(newValue)}
+          options={displayedLanguages}
+          renderInput={(params) => (
+            <TextField {...params} label='Your native language' />
+          )}
+          value={selectedLanguage}
+        />
+
```

Comment:
it would be better to wrap this in a `<Box>` so that there is not too much space between the text and the autocomplete field

Then add ` sx={{ marginTop: '20px' }}` to `<Autocomplete>` to set the space between text and form

Your design 
![image](https://github.com/user-attachments/assets/62889575-2c34-468a-bc06-547f26a6c9ac)

Figma
![image](https://github.com/user-attachments/assets/204073d1-a8a4-4868-aebf-9b05c25e0fd2)